.github/workflows/kcov.yml

```diff
@@ -0,0 +1,35 @@
+name: Kcov test
+on:
+  [push, pull_request]
+jobs:
+  kcov-test:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Update the system
+        run: |
+          sudo apt-get update -y
+
+      - name: Install dependencies
+        run: |
+          sudo apt-get install -y kcov shunit2 bc sqlite3 bsdmainutils
```

Comment:
Maybe you need to add `git dash`

Reply:
Hm, you're right, git isn't pre-installed, but dash is actually available in the base image.

Reply:
I'm trying with dash now, for my last attempt, but I can't test this locally unfortunately as the local runner always runs as root then breaks other tests unrelated to what is happening in here.